.github/workflows/build.yml

```diff
@@ -27,10 +28,33 @@ jobs:
       - name: Run build
         run: yarn build
 
+  # Test Starbase base container image build
```

Comment:
I wonder if it would be helpful to actually have a CI test that validates that Starbase can run from inside of the CI's built base image. This would help ensure that we don't introduce regressions into the Docker image.

Reply:
Not a bad idea at all. How extensively do you reckon the test should verify that? I.e. actually run some starbase graph integrations or just verify that `yarn starbase setup` etc. works?